app/code/Bulbulatory/Recomendations/Controller/Index/Config.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
Żeby ten kontroler działał potrzebujesz pliku XML w lokalizacji ` app/code/Bulbulatory/Recommendations/etc/frontend/routes.xml` o treści:

```xml
<?xml version="1.0" encoding="UTF-8" ?>
    <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:App/etc/routes.xsd">
    <router id="standard">
        <route id="recomendations" frontName="recomendations">
            <module name="Bulbulatory_Recomendations"/>
        </route>
    </router>
</config>
```

Więcej na ten temat będzie w zadaniu 8 ale jak chcesz to możesz poczytać już teraz - przykładowy tutorial do kontrolerów: https://www.mageplaza.com/magento-2-module-development/how-to-create-controllers-magento-2.html